app/javascript/components/about-us/AboutUsContributors.js

```diff
@@ -25,6 +25,7 @@ export default function AboutUsContributors() {
             <Row type="flex" justify="space-between" align="middle">
                 <Col span={24}>
                     <h2 className="about-us-contributors-title">Quem Somos</h2>
+                    <p>A comunidade Política Para Todos nasceu no verão de 2019 com o objetivo de promover a participação ativa dos cidadãos nos processos eleitorais em Portugal. É composta por voluntários de várias partes do país e diferentes áreas profissionais. Adotamos princípios de transparência radical na nossa forma de trabalho: toda a comunicação interna está disponível numa plataforma aberta , as reuniões são gravadas e também de acesso livre. A adesão à comunidade é aberta para qualquer cidadão.</p>
```

Comment:
I think text centered here does not resulta. I would suggest to add a new `Row` before this one with the paragraph.